### PR TITLE
Legacy api token ux

### PR DESCRIPTION
**Pull request by jsoref**
This is an attempt to clean up the new `Manage Legacy API Token usage` section.

See [JENKINS-52161](https://issues.jenkins-ci.org/browse/JENKINS-52161) for general context.

### Desired reviewers

@daniel-beck @jglick

### CODE REVIEW

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties [hunk 1]
```diff
@@ -4,24 +4,24 @@ contextMessage=The following users have a legacy API token. \
...
+NumOfUse=# of uses
 NumDaysSinceLastUse=Days since last use
 HasFreshToken=Fresh token?
 HasFreshToken_tooltip=A fresh token is one that was created using the new system \
   after the most recent use of the user''s legacy token. \n\
   Note: the creation date of legacy tokens which have never been used are reset every time Jenkins is restarted, \
-  which means that the date may be inaccurate.
-HasFreshToken_ok_tooltip=A fresh token exist for that user
-HasFreshToken_warning_tooltip=No fresh token exist for that user
+  which means that the date will be inaccurate.
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties]: I liked *may* better (not all tokens, just specific ones), or perhaps *… means that **their** date…*?

**Review by daniel-beck — APPROVED** [review]

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties (hunk 1 above)
Comment (jsoref) [reply core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties]: _... means the date for **those** tokens will be inaccurate_?

Comment (daniel-beck) [reply core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties]: Or that 👍

**Comment by jsoref** [comment]
One final change:
```jenkins | Jenkins Account | Legacy Token | 0 | 155 | 0 | ✅ | ✅``` 
The tooltips for the two ✅ items used `that`, but the appropriate word is `this`

**Review by Wadeck — APPROVED** [review]
Perfect, thank you 👍

**Review by daniel-beck — APPROVED** [review]

**Comment by daniel-beck** [comment]
@Wadeck Does https://github.com/jenkinsci/jenkins/pull/3525 being merged change anything about this PR, or are they independent?

**Comment by Wadeck** [comment]
@daniel-beck complementary :)

**Pull request merged**